cradle-core/src/test/java/com/exactpro/cradle/utils/CompressionTypeTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.utils;
+
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+import static com.exactpro.cradle.utils.CompressionType.LZ4;
+import static com.exactpro.cradle.utils.CompressionType.ZLIB;
+import static org.testng.Assert.assertEquals;
+
+public class CompressionTypeTest {
```

Comment:
I think we should also add tests for our `CompressionType.decompress` function that try to identify the type of compression under the hood

Reply:
They are tested in the lines
- assertEquals(bytes, CompressionType.decompressData(compressed), "Decompress via the decompressData method");